packages/cadl-ranch-specs/http/resiliency/srv-driven-1/main.tsp

```diff
@@ -1,100 +1,148 @@
 import "@typespec/http";
+import "@typespec/versioning";
 import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
 
-@doc("DPG Swagger, this is the initial swagger a service could do")
-@scenarioService("/resiliency/servicedriven1")
-namespace Resiliency.ServiceDriven1;
 using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+using TypeSpec.Versioning;
 
-model PostInput {
-  url: string;
-}
+@versioned(Versions)
+@doc("""
+  Test that we can grow up a service spec and service deployment into a multi-versioned service with full client support.
+  """)
+@client({
+  name: "ResiliencyServiceDrivenClient",
+})
+@service
+@server(
+  "http://localhost:3000/resiliency/service-driven/client:v1/service:{serviceDeploymentVersion}/api-version:{apiVersion}",
+  "Testserver endpoint",
+  {
+    @doc("Pass in either 'v1' or 'v2'. This represents a version of the service deployment in history. 'v1' is for the deployment when the service had only one api version. 'v2' is for the deployment when the service had api-versions 'v1' and 'v2'.")
```

Comment:
should we use `v1` and `v2` for the versioning of the client, the service deployment and api version? @lmazuel is also wondering if we want to do more accurate semantic versioning / using dates for api version versioning. I'm ok with any versioning with de for `api-version`, but I'm thinking if we want to do `old` and `new` for service deployment version and client